doc/source/serve/index.rst

```diff
@@ -206,7 +206,7 @@ When should I use Ray Serve?
 
 Ray Serve is a flexible tool that's easy to use for deploying, operating, and monitoring Python-based machine learning applications.
 Ray Serve excels when you want to mix business logic with ML models and scaling out in production is a necessity. This might be because of large-scale batch processing
-requirements or because you want to scale up a model pipeline consisting of many individual models with different performance properties.
+requirements or because you want to scale up a model deployment graph consisting of many individual models with different performance properties.
```

Comment:
```suggestion
requirements or because you want to scale up a deployment graph consisting of many individual models with different performance properties.
```